# Población, muestra y clasificación de variables

## Objetivos:

Comprender y aplicar los conceptos de población, muestra y tipos de variables (cualitativas y cuantitativas) en contextos aplicados.

*Nota: Este laboratorio está diseñado para desarrollar razonamiento estadístico. Evita el uso de herramientas de IA para responder las preguntas a menos que esto sea explícitamente sugerido* 🤖. *En su lugar, enfócate en aplicar los conceptos revisados.*

## Actividad 1

Una universidad realizó una encuesta a estudiantes de carreras de informática para conocer su comportamiento digital. Se consultaron variables como edad, red social preferida, horas promedio en redes al día, y nivel de satisfacción con su conexión a internet.

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/15PDo62bGOsSoQUg2qIT8OOqe5Y1tJ53T/export?format=xlsx
">aquí</a>**.



1. ¿Cuál es la población del estudio?

La población corresponde a los estudiantes de carreras de informática de la universidad.

2. ¿Cuál es la muestra del estudio?

In [1]:
import pandas as pd

#Creación de DataFrame con base de datos.
df = pd.read_excel('Encuesta_uso_redes.xlsx', sheet_name = 'Base 1')

FileNotFoundError: [Errno 2] No such file or directory: 'Encuesta_uso_redes.xlsx'

In [ ]:
muestra = df.shape[0] #Número de filas
muestra

300

La muestra del estudio son 300 estudiantes de carreras de informática de la universidad.

3. Muestra las primeras cinco filas y las últimas cinco filas del DataFrame.

In [ ]:
#Muestra las primeras 5 filas de DF
df.head(5)

Edad Red Social Preferida  Horas Promedio Diario  \
0    24                    X                    1.0   
1    32             Snapchat                    2.2   
2    28             LinkedIn                    1.8   
3    25                    X                    1.0   
4    24             Snapchat                    2.5   

  Nivel Satisfacción Internet  
0                     Neutral  
1                  Satisfecho  
2                Insatisfecho  
3                     Neutral  
4            Muy insatisfecho

In [ ]:
#Muestra las últimas 5 filas de DF
df.tail(5)

Edad Red Social Preferida  Horas Promedio Diario  \
295    25             LinkedIn                    4.1   
296    29               TikTok                    3.4   
297    29                    X                    5.7   
298    22                    X                    0.7   
299    24               TikTok                    7.2   

    Nivel Satisfacción Internet  
295                  Satisfecho  
296            Muy insatisfecho  
297                  Satisfecho  
298                Insatisfecho  
299              Muy satisfecho

4. Clasifica cada una de las variables presentes en el estudio.

**Edad**: Cuantitativa continua

**Red social preferida**: Cualitativa nominal

**Horas promedio diario**: Cuantitativa continua

**Nivel de satisfacción**: Cualitativa ordinal

5. ¿Por qué es importante distinguir entre tipos de variables al planificar un análisis estadístico?

Identificar la clasificación de las variables es fundamental porque permite planificar qué técnicas estadísticas son adecuadas para cada caso. Por ejemplo, las variables cualitativas pueden ser analizadas mediante tablas de frecuencia y gráficos, mientras que las variables cuantitativas permiten realizar análisis un poco más complejos.

## Actividad 2

Una empresa de telecomunicaciones registró durante un semestre todos las problemas técnicos reportados por sus clientes. Se registraron variables como tipo de problema, tiempo de resolución, región y nivel de gravedad.

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1tgPbbtaf9svlo35Kjz8kf9z8gAu3o0yC/export?format=xlsx
">aquí</a>**.

1. ¿Cuál es la población del estudio?

La población del estudio corresponde a los problemas técnicos que los clientes reportan.

2. ¿Cuál es la muestra del estudio?

In [ ]:
import pandas as pd

#Creación de DataFrame con base de datos.
df2 = pd.read_excel('Problemas_redes.xlsx', sheet_name = 'Base 2')

In [ ]:
muestra2 = df2.shape[0] #Número de filas
muestra2

250

La muestra del estudio corresponde a 250 problemas técnicos que los clientes reportaron.

3. Muestra las primeras cinco filas y las últimas cinco filas del DataFrame.

In [ ]:
#Muestra las primeras 5 filas de DF
df2.head(5)

Tipo de problema  Tiempo de resolución (minutos)  Region Gravedad
0         Lentitud                              17  Centro     Baja
1    Interferencia                              39  Centro    Media
2         Lentitud                              15  Centro     Baja
3         Lentitud                              37   Norte    Media
4     Caída de red                              29  Centro    Media

In [ ]:
#Muestra las últimas 5 filas de DF
df2.tail(5)

Tipo de problema  Tiempo de resolución (minutos)  Region Gravedad
245     Caída de red                              32     Sur    Media
246         Lentitud                              27  Centro    Media
247     Caída de red                              33  Centro    Media
248     Caída de red                              62   Norte     Alta
249    Interferencia                              37   Norte    Media

4. Clasifica cada una de las variables presentes en el estudio.

**Tipo de problema**: Cualitativa nominal

**Tiempo de resolución (minutos)**: Cuantitativa continua

**Región**: Cualitativa nominal

**Gravedad**: Cualitativa ordinal

5. Considera el siguiente gráfico que muestra la relación entre la gravedad de un problema y el tiempo promedio que tardan en resolverlo, luego responde: ¿Qué relación observas entre el nivel de gravedad del problema y el tiempo promedio que tardan en resolverlo? Justifica tu respuesta con base en el gráfico.

<center>

<img src = "https://i.ibb.co/Z6qvTjnB/Captura-de-pantalla-2025-05-19-172338.png" width = "450"/>

Se puede observar que a medida que aumenta la gravedad del problema, también aumenta el tiempo promedio de resolución. Esto sugiere que los problemas de mayor gravedad tardan, en promedio, más tiempo en solucionarse.

6. ¿Qué problemas podría haber si clasificamos mal una variable al trabajar con datos?

Clasificar mal una variable puede inducir errores graves, ya que podríamos utilizar técnicas estadísticas incorrectas. Esto puede afectar la precisión del análisis y las decisiones que son tomadas con base en los datos.

## Actividad 3

El uso de la inteligencia artificial (IA) en la educación ha emergido como una poderosa herramienta para transformar los métodos de enseñanza en todo el mundo. Según el último Monitor Global de Educación de Ipsos, publicado en el mes de septiembre de 2023, queda claro que la percepción de la importancia de capacitar a los maestros en el uso de la IA varía significativamente de un país a otro. En general, un 65% de los encuestados en 29 países piensan que los docentes deberían recibir capacitación sobre cómo usar la IA en sus estrategias educativas.

Fuente <a href="https://es.statista.com/grafico/30894/encuestados-que-consideran-esencial-que-maestros-se-capaciten-para-usar-ia-en-la-ensenanza/">Statista: Inteligencia artificial: ¿importante en la enseñanza? </a>

<img src = "https://cdn.statcdn.com/Infographic/images/normal/30894.jpeg" width = "500"/>

1. ¿Cuál es la población considerada en este estudio?

La población de este estudio está compuesta por los encuestados que participaron en el Monitor Global de Educación de Ipsos, realizado en septiembre de 2023 en 29 países.

2. ¿Cuál es la muestra considerada en el estudio?

La muestra de este estudio está compuesta por 23.248 encuestados que participaron en el Monitor Global de Educación de Ipsos, realizado en septiembre de 2023 en 29 países.

3. ¿Cuál es la variable presente en el estudio? Piensa en cuáles podrían ser los posibles valores de esta variable y clasifícala.

La variable es **Percepción de la importancia de capacitar a los maestros en el uso de la IA**. Si la pregunta tuviera como respuesta (valores): si, no o a veces. Su clasificación sería cualitativa nominal. Pero si las respuestas fueran en escala como: muy importante hasta nada importante, la variable sería cualitativa ordinal.

4. ¿Qué significa que un país obtenga un menor puntaje?

Significa que es menor la cantidad de encuestados que consideran esencial que los maestros se capaciten para usar la IA en sus métodos de enseñanza.

5. Realice conclusiones que pueda extraer de la observación del gráfico.

Los encuestados de Perú y Sudáfrica destacan la importancia de la capacitación de los maestros en el uso de la inteligencia artificial en sus métodos de enseñanza, a diferencia de los participantes de Estados Unidos y Francia. En estos últimos dos países, menos de la mitad de los encuestados consideran esencial dicha capacitación.

## Actividad 4

En 2022, el mercado de audiolibros estuvo valorado en aproximadamente 5.365 millones de dólares estadounidenses. Se espera que el mercado siga en expansión, con una tasa de crecimiento anual compuesta de 26,3% entre 2023 y 2030, alcanzando un valor de alrededor de 35.000 millones de dólares en el último año.

Fuente  <a href="https://es.statista.com/grafico/30959/encuestados-que-consumieron-audiolibros-en-los-ultimos-doce-meses/">Statista: ¿Quiénes consumen audiolibros? </a>

<img src = "https://cdn.statcdn.com/Infographic/images/normal/30959.jpeg" width = "500"/>

1. ¿Cuál es la población estadística considerada en el estudio?

La población del estudio son los encuestados entre 18 y 64 años de diversos países.

2. ¿Por qué la suma de los porcentajes no es igual a 100%?

La suma de los porcentajes del gráfico no alcanza el 100% debido a que cada barra representa el porcentaje de aquellos que sí consumieron audiolibros en relación con el total de participantes en cada país. En otras palabras, el porcentaje se calcula con respecto al número total de encuestados en cada país que respondieron sobre el consumo de audiolibros, y no sobre el total general de la población.

3. Realice una conclusión breve de la información presentada.

En el transcurso de los últimos doce meses, el 42% de los participantes de la encuesta en China han optado por consumir audiolibros mediante descargas o transmisión en línea. En contraste, en Japón, solo un 8% de los participantes reportaron realizar esta actividad.

## Actividad 5

En una encuesta reciente realizada por Ipsos a más de 23.000 personas de 31 países, se han destacado las preocupaciones más apremiantes de los entrevistados en relación con el sistema de salud de su país.

Fuente <a href="https://es.statista.com/grafico/31156/encuestados-que-creen-que-los-siguientes-son-los-mayores-problemas-del-sistema-sanitario-de-su-pais/">Statista: Percepciones y desafíos de la salud pública </a>

<img src = "https://cdn.statcdn.com/Infographic/images/normal/31156.jpeg" width = "500"/>

1. ¿Cuál es la población considerada en este estudio?

La población corresponde a los adultos de 31 países.

2. ¿Cuál es la muestra considerada en el estudio?

La muestra corresponde a 23.667 adultos de 31 países.

3. Identifique y clasifique la(s) variable(s) presente(s) en el estudio.

La única variable presente en el estudio es la **Percepción de los mayores problemas de salud en sus países**, se clasifica como cualitativa nominal.

4. Realice una conclusión breve que pueda extraer de la observación del gráfico.

Los principales problemas identificados se relacionan con la insuficiencia de personal (45%) y acceso al tratamiento/largos tiempos de espera (46%), siendo seleccionados por casi la mitad de los participantes de la encuesta. Estas preocupaciones se destacan como las más importantes en la encuesta.

Adicionalmente, otro desafío destacado se relaciona con los costos asociados al acceso al tratamiento, mencionado por el 32% de los participantes de la encuesta.